### ISSUE TITLE
Complete training function in LSTM driver

### ISSUE THREAD

**Issue by rmattson1008** [Micky774/ornet#12]
The training function for the LSTM is only a skeleton - it just serves to check if the model runs. It needs added functionality like keeping track of validation loss and training loss.

Additionally, we can choose to terminate training when validation loss stops improving to prevent overfitting. I'd like to discuss if this is a good idea - does the small amount of samples we are working with affect the wisdom of this approach?
